ui-common/src/components/TrafficViewer/EntriesList.tsx

```diff
@@ -148,14 +148,14 @@ export const EntriesList: React.FC<EntriesListProps> = ({
         case "queryMetadata":
           setTruncatedTimestamp(message.data.truncatedTimestamp);
           setQueriedTotal(message.data.total);
-          if (leftOffTop === null) {
-            setLeftOffTop(message.data.leftOff - 1);
+          if (leftOffTop === "") {
+            setLeftOffTop(message.data.leftOff);
```

Comment:
maybe need to decreament the id by one?

Reply:
We can't decrement it because it's a string now. This change is also wrong https://github.com/up9inc/mizu/pull/984/commits/2899414f2b82ed8fdb2e84a7b6aa00f8e8b8f5e0#diff-3a5137adc8ec2619a9214ef04911adfb8b3c3e76da4eb75169ee697b899d580aR58-R59

It's not always a zero padded string that's actually an integer. You can't do arithmetic operations or modifications on the `id` or `leftOff` value inside the front-end anymore.

Reply:
To clarify again, with this PR, the value of `id`, `leftOff` fields or `leftOff` helper argument can be;

- `000000000000000000000042`
- `507f1f77bcf86cd799439011`

Reply:
Ok, so maybe I didn't understand it before. 
Anyway the issue with 1 is something that we don't understand something with the leftoff so I will try to explain it later and see if we can find a good option here (with the latest fixed we did)

Reply:
https://github.com/up9inc/mizu/commit/9e1dccac501c2eae42bf9c76d58c1ebeeef14d3d